__tests__/components/CommunityActivity.test.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { CommunityActivity } from 'components/CommunityActivity';
+import { CommunityAccount } from 'lib/community';
+
+const account: CommunityAccount = {
+  id: '0xe89cb2053a04daf86abaa1f4bc6d50744e57d39e',
+  token: { id: '0', uri: 'some-uri' },
+  categoryScoreChanges: [
+    {
+      id: '0x719952bc373e34fe932a2158a3221e6d4d3d078a694e15de42c9faca33173103-ACTIVITY-1',
```

Comment:
@adamgobes is this correct? I thought we discussed changing the JSON so that the ID is the key and the lookup is easier?